lib/matplotlib/contour.py

```diff
@@ -1764,6 +1769,17 @@ def _initialize_x_y(self, z):
     iterable is shorter than the number of contour levels
     it will be repeated as necessary.
 
+negative_linestyles : {*None*, 'solid', 'dashed', 'dashdot', 'dotted'}, optional
```

Comment:
What is the proper way to break this line to meet line length requirements?

Reply:
`*None* or string, optional` and then list the acceptable strings below?

Reply:
I'm not convinced it should be `*None*`, as `*...*` is used to mark parameter names, not values. Although it is clearly used in contour.py and in a few other locations.

Grepping a bit, a plain `None` seems to be the most common option. (Although `*None*` is used in > 100 locations in the code.)

Reply:
I did the same thing and found mixed results. I am happy to remove *...* if that is the consensus.

I also found a mix of "None or str", and "str or None"...

Reply:
I'm hoping that someone will come in with a consensus statement. :-)

But I'm pretty sure that not using `*None*` will be less controversial than using it. Also, I believe that in the text it should either be a plain None or a double single-backtick one (to render as code), although the former is clearly the most common.

Reply:
I think in rst it's *None*, but not sure about the docstrings.